test/dotnet-new.Tests/New3/GivenThatIWantANewApp.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.IO;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.New3.Tests
+{
+    public class GivenThatIWantANewApp : TestBase
+    {
+        [Fact]
+        public void When_dotnet_new_is_invoked_mupliple_times_it_should_fail()
```

Comment:
Inheriting test classes from `new` with a few abstract setup / object factory methods could be a good way to ensure test parity, if that's helpful.